modules/oadp-installing-dpa-1-3.adoc

```diff
@@ -0,0 +1,392 @@
+
+** `Secret` with a custom name for the backup location. You add this `Secret` to the `DataProtectionApplication` CR.
+** `Secret` with the default name, `{credentials}`, for the snapshot location. This `Secret` is not referenced in the `DataProtectionApplication` CR.
+endif::[]
+ifdef::installing-oadp-aws[]
+* If the backup and snapshot locations use different credentials, you must create a `Secret` with the default name, `{credentials}`, which contains separate profiles for the backup and snapshot location credentials.
+endif::[]
++
+[NOTE]
+====
+If you do not want to specify backup or snapshot locations during the installation, you can create a default `Secret` with an empty `credentials-velero` file. If there is no default `Secret`, the installation will fail.
+====
+
+.Procedure
+
+. Click *Operators* -> *Installed Operators* and select the OADP Operator.
+. Under *Provided APIs*, click *Create instance* in the *DataProtectionApplication* box.
+
+. Click *YAML View* and update the parameters of the `DataProtectionApplication` manifest:
+ifdef::installing-oadp-aws[]
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: <dpa_sample>
+  namespace: openshift-adp <1>
+spec:
+  configuration:
+    velero:
+      defaultPlugins:
+        - openshift <2>
+        - aws
+      resourceTimeout: 10m <3>
+    nodeAgent: <4>
+      enable: true <5>
+      uploaderType: kopia <6>
+      podConfig:
+        nodeSelector: <node_selector> <7>
+  backupLocations:
+    - name: default
+      velero:
+        provider: {provider}
+        default: true
+        objectStorage:
+          bucket: <bucket_name> <8>
+          prefix: <prefix> <9>
+        config:
+          region: <region>
+          profile: "default"
+        credential:
+          key: cloud
+          name: {credentials} <10>
+  snapshotLocations: <11>
+    - name: default
+      velero:
+        provider: {provider}
+        config:
+          region: <region> <12>
+          profile: "default"
+----
+<1> The default namespace for OADP is `openshift-adp`. The namespace is a variable and is configurable.
+<2> The `openshift` plugin is mandatory.
+<3> Specify how many minutes to wait for several Velero resources before timeout occurs, such as Velero CRD availability, volumeSnapshot deletion, and backup repository availability. The default is 10m.
+<4> The administrative agent that routes the administrative requests to servers.
+<5> Set this value to `true` if you want to enable `nodeAgent` and perform File System Backup.
+<6> Enter `kopia` or `restic` as your uploader. You cannot change the selection after the installation. The `nodeAgent` deploys a daemon set, which means that the `nodeAgent` pods run on each working node. You can configure File System Backup by adding `spec.defaultVolumesToFsBackup: true` to the `Backup` CR.
```

Comment:
@weshayutin @kaovilai @stillalearner 
Should we include here a sentence saying "You must use Kopia for the Built-in DataMover."?